<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>989</width>
    <height>722</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindh3_table_right_clickow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>971</width>
      <height>431</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="documentMode">
     <bool>false</bool>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>주문</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_8">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>181</width>
        <height>251</height>
       </rect>
      </property>
      <property name="title">
       <string>수동주문</string>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>56</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>계좌</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>45</x>
         <y>15</y>
         <width>120</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>56</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>주문</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_2">
       <property name="geometry">
        <rect>
         <x>45</x>
         <y>45</y>
         <width>120</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>신규매수</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>신규매도</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>매수취소</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>매도취소</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>56</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>종목</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>45</x>
         <y>75</y>
         <width>120</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_3">
       <property name="geometry">
        <rect>
         <x>45</x>
         <y>135</y>
         <width>120</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>지정가</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>시장가</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>56</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>종류</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>56</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>수량</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox">
       <property name="geometry">
        <rect>
         <x>45</x>
         <y>165</y>
         <width>120</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>1000000</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>56</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>가격</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_2">
       <property name="geometry">
        <rect>
         <x>45</x>
         <y>195</y>
         <width>120</width>
         <height>22</height>
        </rect>
       </property>
       <property name="prefix">
        <string/>
       </property>
       <property name="maximum">
        <number>10000000</number>
       </property>
       <property name="singleStep">
        <number>100</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>220</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>현금주문</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="geometry">
        <rect>
         <x>45</x>
         <y>105</y>
         <width>120</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(170, 255, 255);</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_7">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>220</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>잔고조회</string>
       </property>
      </widget>
     </widget>
     <widget class="QTableWidget" name="tableWidget_4">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>10</y>
        <width>761</width>
        <height>251</height>
       </rect>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <column>
       <property name="text">
        <string>종목코드</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>종목명</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>현재가</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>전일대비 (%)</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>장부가</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>보유수량</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>매입금액</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>평가금액</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>수익률 (%)</string>
       </property>
      </column>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>951</width>
        <height>101</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>자동 매매 시작</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_9">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>270</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>자동 매매 중지</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>잔고조회</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>951</width>
        <height>371</height>
       </rect>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <column>
       <property name="text">
        <string>종목코드</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>종목명</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>현재가</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>전일대비 (%)</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>장부가</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>보유수량</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>매입금액</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>평가금액</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>수익률 (%)</string>
       </property>
      </column>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>0</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pieButton">
         <property name="text">
          <string>분포차트</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>잔고조회</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>전략검색</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>331</width>
        <height>401</height>
       </rect>
      </property>
      <property name="title">
       <string>전략리스트</string>
      </property>
      <widget class="QListWidget" name="listWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>311</width>
         <height>331</height>
        </rect>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>311</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButton_4">
          <property name="text">
           <string>전략리스트 가져오기</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_6">
          <property name="text">
           <string>Clear all</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>10</y>
        <width>601</width>
        <height>401</height>
       </rect>
      </property>
      <property name="title">
       <string>검색 결과</string>
      </property>
      <widget class="QListView" name="listView">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>241</width>
         <height>301</height>
        </rect>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>261</width>
         <height>331</height>
        </rect>
       </property>
       <property name="title">
        <string>전략 검색 조건</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_7">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>20</y>
         <width>281</width>
         <height>371</height>
        </rect>
       </property>
       <property name="title">
        <string>전략 검색 종목</string>
       </property>
       <widget class="QTableWidget" name="tableWidget_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>261</width>
          <height>341</height>
         </rect>
        </property>
        <column>
         <property name="text">
          <string>종목코드</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>종목명</string>
         </property>
        </column>
       </widget>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>21</y>
         <width>251</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>검색 종목 개수 : </string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <zorder>layoutWidget</zorder>
      <zorder>groupBox_7</zorder>
      <zorder>groupBox_2</zorder>
      <zorder>listView</zorder>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>160</y>
        <width>31</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>&gt;&gt;</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>211</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>접속정보</string>
    </property>
    <widget class="QPushButton" name="login_Creon">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>27</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>로그인</string>
     </property>
    </widget>
    <widget class="QLabel" name="account_numbox">
     <property name="geometry">
      <rect>
       <x>79</x>
       <y>61</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>61</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>계좌번호 :</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>27</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>로그아웃</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>549</y>
      <width>971</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>로그</string>
    </property>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>951</width>
       <height>111</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>751</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>계좌정보</string>
    </property>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>731</width>
       <height>61</height>
      </rect>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <column>
      <property name="text">
       <string>종목수량</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>총매입</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>총평가</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>총손익</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>총수익률(%)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>예수금</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>추정자산</string>
      </property>
     </column>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>25</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>989</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>tableWidget_2</sender>
   <signal>customContextMenuRequested(QPoint)</signal>
   <receiver>MainWindow</receiver>
   <slot>h3_table_right_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1155</x>
     <y>537</y>
    </hint>
    <hint type="destinationlabel">
     <x>1155</x>
     <y>627</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>h3_table_right_click()</slot>
 </slots>
</ui>
